.golangci.yml

```
@@ -25,7 +25,24 @@
-
+    gofumpt:
+      # Module path which contains the source code being formatted.
+      # Default: ""
+      module-path: github.com/coolguy1771/wastebin
+    gci:
+        # Section configuration to compare against.
+        # Section names are case-insensitive and may contain parameters in ().
+        # The default order of sections is `standard > default > custom > blank > dot > alias > localmodule`.
+        # If `custom-order` is `true`, it follows the order of `sections` option.
+        # Default: ["standard", "default"]
+        sections:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)